packages/devtools_app/lib/src/screens/memory/memory_instance_tree_view.dart

```diff
@@ -45,42 +44,43 @@ class InstanceTreeViewState extends State<InstanceTreeView>
     super.didChangeDependencies();
 
     final newController = Provider.of<MemoryController>(context);
-    if (newController == controller) return;
-    controller = newController;
+    if (_controllerInitialized && newController == _controller) return;
+    _controller = newController;
+    _controllerInitialized = true;
 
     cancelListeners();
 
     // TODO(terry): setState should be called to set our state not change the
     //              controller. Have other ValueListenables on controller to
     //              listen to, so we don't need the setState calls.
     // Update the chart when the memorySource changes.
-    addAutoDisposeListener(controller.selectedSnapshotNotifier, () {
+    addAutoDisposeListener(_controller.selectedSnapshotNotifier, () {
       setState(() {
-        controller.computeAllLibraries(rebuild: true);
+        _controller.computeAllLibraries(rebuild: true);
       });
     });
   }
 
   @override
   Widget build(BuildContext context) {
-    controller.instanceFieldsTreeTable = TreeTable<FieldReference>(
-      dataRoots: controller.instanceRoot,
+    _controller.instanceFieldsTreeTable = TreeTable<FieldReference>(
+      dataRoots: _controller.instanceRoot!,
       columns: columns,
       treeColumn: treeColumn,
       keyFactory: (typeRef) => PageStorageKey<String>(typeRef.name),
       sortColumn: columns[0],
       sortDirection: SortDirection.ascending,
     );
 
-    return controller.instanceFieldsTreeTable;
+    return _controller.instanceFieldsTreeTable!;
   }
 }
 
 class _FieldTypeColumn extends TreeColumnData<FieldReference> {
   _FieldTypeColumn() : super('Type');
 
   @override
-  dynamic getValue(FieldReference dataObject) =>
+  Object? getValue(FieldReference dataObject) =>
```

Comment:
non-nullable. Also should this be returning a String? it seems most other columns return a string value for their `getValue` method

Reply:
done